boat-scaffold/src/main/templates/boat-angular/tsconfig.mustache

```diff
@@ -23,6 +28,17 @@
     ],
     "filesGlob": [
         "./model/*.ts",
-        "./api/*.ts"
-    ]
+        "./api/*.ts",
+        "**/*.ngfactory.ts",
```

Comment:
are these extra ones really needed?

Reply:
This is what's in [the default tsconfig used by ng-packagr](https://github.com/ng-packagr/ng-packagr/blob/v12.2.6/src/lib/ts/conf/tsconfig.ngc.json)

i guess as we're not generating any files that match these patterns they're not strictly needed.

Reply:
the plan is to keep tsconfig like the default from ng-packagr as much as possible. But we will create a ticket to make it extend instead of copying everything